bin/desi_tsnr_afterburner

```diff
@@ -68,8 +68,6 @@ def parse(options=None):
                         help = 'Only compute the alpha for tsnr.')
     parser.add_argument('--nproc', type = int, default = 1,
                         help = 'Multiprocessing.')
-    parser.add_argument('--efftime-config', type = str, default = None, required=False,
-                        help = 'Use this config file instead of default data/tsnr/tsnr-efftime.yaml')
```

Comment:
Documenting a chat with @julienguy : this option is removed because it previously enabled the ability to have this config file inconsistent with the TSNR templates in desimodel.  Now the TSNR files in desimodel themselves contain all the information for how to interpret them in a self-consistent manner.